script/release/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.17.7 AS build
+
+ARG CONTAINERD_VERSION=1.5.7
+ARG NYDUS_VERSION=v2.0.0-rc.6
+
+ARG CONTAINERD_LINUX_AMD64_SHA256SUM="109fc95b86382065ea668005c376360ddcd8c4ec413e7abe220ae9f461e0e173"
+ARG NYDUS_LINUX_AMD64_SHA256SUM="caca4be643ee11e74ff7be8409fc3bc0fff231395f324f41baadc3737a1156ec"
+
+# Install acceld
+ADD ./ /accel
```

Comment:
nit: `COPY` should be used when possible https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
Fixed, thanks!